media-sound/ncmpc/ncmpc-0.30.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit meson ninja-utils
+
+DESCRIPTION="A ncurses client for the Music Player Daemon (MPD)"
+HOMEPAGE="https://www.musicpd.org/clients/ncmpc/ https://github.com/MusicPlayerDaemon/ncmpc"
+SRC_URI="http://www.musicpd.org/download/${PN}/${PV%.*}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="artist-screen chat-screen debug doc +help-screen key-screen lirc lto lyrics-screen mouse nls outputs-screen search-screen song-screen"
+
+RDEPEND=">=dev-libs/glib-2.30:2
+	>=media-libs/libmpdclient-2.9
+	sys-libs/ncurses:0=[unicode]
+	lirc? ( app-misc/lirc )"
+DEPEND="${RDEPEND}
+	app-arch/xz-utils
+	virtual/pkgconfig"
+
+DOCS=( AUTHORS NEWS README.rst doc/config.sample doc/keys.sample )
+
+meson_use() {
+	usex "$1" true false
+}
+
+src_prepare() {
+	eapply ${FILESDIR}/fix_enqueue_from_search.patch
+	eapply ${FILESDIR}/fix_gcc_warning.patch
+	eapply ${FILESDIR}/restore_progress_bar_in_colorless_mode.patch
+	eapply_user
+}
+
+src_configure() {
+	local myconf=(
+		-Db_ndebug=$(usex debug false true)
```

Comment:
don't do this, we don't randomly change build types

Reply:
this is something that should be decided at the eclass level, not on a per-package basis